app/templates/app/all-installments.html

```diff
@@ -0,0 +1,100 @@
+{% extends 'base.html' %}
+<!--{% load bootstrap4 %}-->
+<!--{% bootstrap_css %}-->
+{% block title %} Installments {% endblock title %}
+{% block content %}
+<html>
+  <head>
+    <link rel="stylesheet" href="../../static/css/installment_table.css">
```

Comment:
This could be resolved using a static template tag? https://docs.djangoproject.com/en/1.11/howto/static-files/